src/main/java/org/learn_java/bot/commands/free/NewFree.java

```diff
@@ -0,0 +1,42 @@
+package org.learn_java.bot.commands.free;
+
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.vdurmont.emoji.EmojiManager;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.managers.ChannelManager;
+import org.springframework.stereotype.Component;
+
+@Component
+public class NewFree extends Command {
```

Comment:
What is the point of this class?

Reply:
I guess ill remove the freeing stuff cause of discord's API, and i cant' be bothered to implement a queue